docs/main/extendable-workflow/README.md

```diff
@@ -25,10 +25,14 @@ A `secret` containing the instance credentials must exists as well.
 See https://www.parodos.dev/orchestrator-helm-chart/postgresql on how to install a PostgreSQL instance. Please follow the section detailing how to install using helm. In this document, a `secret` holding the credentials is created.
 
 ## Prerequisites 
-Set `TARGET_NS` to the target namespace:
+* Set `TARGET_NS` to the target namespace:
 ```console
 TARGET_NS=sonataflow-infra
 ```
+
+* The custom UI widget shall be installed:
```

Comment:
Why? `shall` is more formal
> In some documents and official situations you can find ‘shall’ used to describe formal obligations.

Source: https://www.wallstreetenglish.com/exercises/the-difference-between-must-have-to-shall-need-and-may

Reply:
To me it reads like "the widget will be installed, but not by the user, it will be installed by itself." Note that in the examples in the link you attached, "shall" comes after the party who is committing the obligation ("Employees shall provide a medical certificate for sick leave.").

Reply:
mmmh, I am neither a native nor have a degree in english so I won't fight you on this one :)